reference/display.md

```diff
@@ -175,10 +175,7 @@ void wb_display_detach_camera(WbDeviceTag tag);
 
 **Description**
 
-These functions are used to attach/detach a camera to a display. If a camera is attached to a display, as soon as a new image is generated by the camera it is efficiently copied in the display. This is useful to simulate mirrors for example.
-
-> **note**:
-When a camera is attached to a display it is not possible any more to draw on this display.
+These functions are used to attach/detach a camera to a display. When a camera is attached to a display, the camera images are efficiently copied to the background of the display. This is useful to simulate for example mirrors or head-up display (HUD).
```

Comment:
Yes, this is maybe a bad example. What I would like to say is something display on the top of a camera rendering, but I don't have good names for this. Maybe it's just about to tell that this is designed for drawing something on the top of a camera node, such as debug information..